### PR TITLE
Correct suggestions

### PR DESCRIPTION
**Pull request by datmau**
This PR is for solve the review suggestions.
- [✔ ] Green travis build
- [ ✔] Unit test

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1 +1 @@
-rootProject.name = 'bpmorganizationsapi'
+rootProject.name = 'organizations'
```
Comment (enriquezrene) [inline settings.gradle]: bpm-organizations-api

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+import org.springframework.http.ResponseEntity;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BpmOrganizationsApiApplicationTests {
+
```
Comment (enriquezrene) [inline src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: All the tests need to be reviewed.

**Review by enriquezrene — APPROVED** [review]

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java (hunk 2 above)
Comment (datmau) [reply src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: @enriquezrene Please Check the new tests and ask me if these are correct.

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    @Test
+    public void whenAOrganizationIsFoundThenReturn200() {
+        Organization organizationFound = mock(Organization.class);
+        String organizationFoundId = "id";
+        when(organizationRepository.findById(organizationFoundId)).thenReturn(Optional.of(organizationFound));
+
+        ResponseEntity response = organizationController.getOrganization(organizationFoundId);
+
+
+        verify(organizationRepository).findById(organizationFoundId);
```
Comment (enriquezrene) [inline src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: We don't need many empty lines here, one is enough.

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        String id = "id";
+        when(organizationRepository.findById(id)).thenReturn(Optional.empty());
+
+        ResponseEntity deleteResponse = organizationController.deleteOrganization(id);
+
+        assertEquals(HttpStatus.NOT_FOUND, deleteResponse.getStatusCode());
+    }
+
+    @Test
+    public void whenAOrganizationIsDeletedAndIsFoundedThenReturnEmpty(){
```
Comment (enriquezrene) [inline src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: Founded does not exist, I think you mean Found

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    @Test
+    public void whenANotFoundOrganizationIsUpdatedThenReturn400() {
+
+        Organization organizationToUpdate = mock(Organization.class);
+        when(organizationRepository.findById(Mockito.anyString())).thenReturn(Optional.empty());
+
+        ResponseEntity<?> updatedOrganizationResponse = organizationController.updateOrganization(Mockito.anyString(), organizationToUpdate);
+
+        assertEquals(HttpStatus.NOT_FOUND, updatedOrganizationResponse.getStatusCode());
+
```
Comment (enriquezrene) [inline src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: Remove empty lines

**Review by enriquezrene — CHANGES_REQUESTED** [review]

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java (hunk 3 above)
Comment (datmau) [reply src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: Done

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java (hunk 4 above)
Comment (datmau) [reply src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: I will be more careful. It's done

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java (hunk 5 above)
Comment (datmau) [reply src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: Done.

File: src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+import org.springframework.http.ResponseEntity;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BpmOrganizationsApiApplicationTests {
```
Comment (enriquezrene) [inline src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java]: This class should be renamed as OrganizationControllerTest, you're testing the class OrganizationController not the class BpmOrganizationsApiApplication.

**Review by enriquezrene — CHANGES_REQUESTED** [review]
Change the test class name and you're done. 

The test class should be renamed as OrganizationControllerTest, you're testing the class OrganizationController not the class BpmOrganizationsApiApplication.

**Review by enriquezrene — APPROVED** [review]

**Pull request merged**